config/manifest/deployment.yaml

```diff
@@ -127,6 +130,10 @@ spec:
             timeoutSeconds: 5
             failureThreshold: 4
             successThreshold: 1
+          # Failing to provide a writable $HOME can cause TUF client initialization to panic
+          volumeMounts:
+          - mountPath: /.sigstore
```

Comment:
Should we put this under `/home/kyverno` and allow TUF to create the ".sigstore"? Would that require us to set the $HOME as indicated in the comment above?